matryoshka_tester/trigger_test.py

```diff
@@ -0,0 +1,119 @@
+import json
+from os import environ
+from typing import Dict, Literal, List, TypedDict, Union
+
+import pika
+import requests
+
+from matryoshka_tester.parse_data import containers
+
+
+class PublishBodyBase(TypedDict):
+    #: name of the project which' publishing state changed
+    project: str
+    #: repository which' publishing state changed
+    repo: str
+
+
+class PacktrackBody(PublishBodyBase):
+    #: some id identifying something
+    payload: str
+
+
+class PublishStateBody(PublishBodyBase):
+    #: the new state of the project + repository
+    state: Literal[
+        "unknown",
+        "broken",
+        "scheduling",
+        "blocked",
+        "building",
+        "finished",
+        "publishing",
+        "published",
+        "unpublished",
+    ]
+
+
+class PublishedBody(PublishBodyBase):
+    #: some internal (=undocumented) build ID
+    buildid: str
+
+
+connection = pika.BlockingConnection(
```

Comment:
I don't know why, but I would move all of this into an init function, which takes certain parameters, so that it's easily unit-testable.

Next to this, if we have another way to trigger_tests in the future that doesn't depend on mqtt, we won't do those unnecessary initializations.

Reply:
I have moved this into the function `RabbitMQConnection.connect_channel_to_bus`